tests/libdv/dv_builder.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Maybe `tests/libdv/dv_builder.go` should be kept more similar to `tests/libvmi/vmi.go`? E.g. renaming it to `tests/libdv/dv.go`?